# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/ipdl/ipdl/direct_call.py
# Commit: c684f79d8c67
# Full Hash: c684f79d8c673f21179fce00358d70845a9692c0
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Rename PWebrtcProxyChannel to PWebrtcTCPSocket. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45098
# ==============================================================================

diff -r 82af7b115f60 -r c684f79d8c67 ipc/ipdl/ipdl/direct_call.py
--- a/ipc/ipdl/ipdl/direct_call.py	Tue Sep 17 18:15:23 2019 +0000
+++ b/ipc/ipdl/ipdl/direct_call.py	Tue Sep 17 18:15:41 2019 +0000
@@ -560,8 +560,8 @@
     ("PSpeechSynthesisRequest", "parent"),
     ("PStunAddrsRequest", "child"),
     ("PStunAddrsRequest", "parent"),
-    ("PWebrtcProxyChannel", "child"),
-    ("PWebrtcProxyChannel", "parent"),
+    ("PWebrtcTCPSocket", "child"),
+    ("PWebrtcTCPSocket", "parent"),
 
     # .h includes something that's a LOCAL_INCLUDE
     ("PBackgroundLocalStorageCache", "child"),